### ISSUE TITLE
Unable to Manually Log In When Linking Accounts

### ISSUE THREAD

**Issue by AlexSneedMiller** [actions-on-google/smart-home-nodejs#46]
When I go to link my service via the Google Home app  on iOS or Android the 'browser' seems to have the credentials cached and does not allow me to enter a username and password. It immediately  grabs a token and then calls my SYNC endpoint. 

Is there a way to prevent this? One of my accounts isn't even working because the username and password are no longer accurate and the Google Home app thinks that it should cache the username and password automatically log in.

**Comment by Fleker** [comment]
If you have previously logged in to an endpoint and your OAuth credentials haven't expired, it'll resume the connection. You can change the endpoint or expire the OAuth tokens to stop the app from authenticating.

**Comment by AlexSneedMiller** [comment]
I tried to use a different auth endpoint, which returned a 404 because I couldn't actually GET the endpoint that I supplied, which makes sense. Then I changed the endpoint back to the original endpoint and it still remembered my username and password. I can see in my logs that the token is changing each time I link and unlink my account, which makes me believe that the Google Home app is logging me in automatically from the login UI.
